jsdoc/assets/static/jscad-docs.css

```diff
@@ -0,0 +1,187 @@
+body {
```

Comment:
HA HA HA... you really don't like the default colors.

Reply:
it's, uh, well. No. No I don't. Not since 1995.